object-group-game/Database/User.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System.Linq;
+
+namespace object_group_game.Database
+{
+	class User
+	{
+		// Checks whether a login with provided username exists
+		public static bool UserExists (string username)
+		{
+			using (var context = new Database.UserContext())
+			{
+				return context.Login.Where(l => l.Username.Equals(username)).FirstOrDefault() != null;
+			}
+		}
+
+		// Checks whether there is a login with provided username and password
+		public static bool CorrectPassword (string username, string password)
+		{
+			using (var context = new Database.UserContext())
+			{
+				return context.Login.Where(l => l.Username.Equals(username) && l.Password.Equals(password)).FirstOrDefault() != null;
+			}
+		}
+
+		// Creates a new login inside the database
+		// Assumes the username is checked for availability
+		public static void NewLogin (string username, string password)
+		{
+			using (var context = new Database.UserContext())
+			{
+				context.Login.Add(new Login() {
+					Username = username,
+					Password = password
+				});
+
+				context.SaveChanges();
+			}
+		}
+	}
+}
```

Comment:
This code should be put into Auth/Authentication.cs class. Not sure what you meant by Login table in DB as we shouldn't need any Login classes/table entries. Everything user/account related should be in Player class and table Players.